src/common/ant.ts

```diff
@@ -278,6 +280,85 @@ export class AoANTReadable implements AoANTRead {
     if (strict) parseSchemaResult(z.number(), balance);
     return balance;
   }
+
+  /**
+   * @returns {Promise<AoANTHandler[]>} The handlers of the ANT.
+   * @example
+   * Get the handlers of the ANT.
+   * ```ts
+   * const handlers = await ant.getHandlers();
+   * ```
+   */
+  async getHandlers(): Promise<AoANTHandler[]> {
+    const info = await this.getInfo();
+
+    return (info.Handlers ?? info.HandlerNames) as AoANTHandler[];
```

Comment:
`getInfo()` returns `AoANTInfo`. if does the inferred type of AoAntInfo return `string[]` for Handlers the way it's setup? Any way to avoid the cast? Not asking to change, just curious if it's required based on how were using zod to generate the types.

Reply:
We might be able to if we switch to a tuple schema but I am uncertain it gives us a clean type.